Data Model/mainProperties.js

```diff
@@ -0,0 +1,153 @@
+export const data =
+    [
+        {
+            id: null,
+            type: 'property',
+            datatype: 'wikibase-item',
+            labels: {
+                pl: 'Typ',
+                en: 'Type'
+            },
+            descriptions: {
+                pl: null,
+                en: null
+            }
+        },
+        {
+            id: null,
+            type: 'property',
+            datatype: 'wikibase-item',
+            labels: {
+                pl: 'miejscowość',
+                en: 'locality'
+            },
+            descriptions: {
+                pl: null,
+                en: null
+            }
+        },
+        {
+            id: null,
+            type: 'property',
+            datatype: 'wikibase-item',
+            labels: {
+                pl: 'dekanat',
+                en: 'deanery'
+            },
+            descriptions: {
+                pl: null,
+                en: null
+            }
+        },
+        {
+            id: null,
+            type: 'property',
+            datatype: 'wikibase-item',
+            labels: {
+                pl: 'diecezja',
+                en: 'diocese'
+            },
+            descriptions: {
+                pl: null,
+                en: null
+            }
+        },
+        {
+            id: null,
+            type: 'property',
+            datatype: 'time',
+            labels: {
+                pl: 'data',
+                en: 'date'
+            },
+            descriptions: {
+                pl: null,
+                en: 'Date may be used to express temporal information at any level of granularity. Recommended best practice is to use an encoding scheme, such as the W3CDTF profile of ISO 8601 [W3CDTF].'
```

Comment:
As I understand it, it will be possible to insert the date in the year format, e.g. 1461, it remains problematic to specify approximations, e.g. the second half of the 16th century, wikidata uses the name "point in time" (https://www.wikidata.org/wiki/Property:P585), so they also applied in wikihum (https://wikihum.lab.dariah.pl/wiki/Property:P40). In CIDOC it is different (http://www.cidoc-crm.org/cidoc-crm/E50_Date).

Reply:
Currently not. But for now it should be enough, because it is wikibase "point in time". Compare with https://www.wikidata.org/wiki/Help:Data_type#Time and the table below. Time here is a "point in time"
![CleanShot 2024-01-18 at 17 18 23](https://github.com/ontohgis/wikikul/assets/92399615/239a83e7-afcb-4948-add8-c31cfa4d28ae)

Finally it's allows a lot.